includes/class-wc-payments-db.php

```diff
@@ -110,4 +110,24 @@ private function order_id_from_meta_key_value( $meta_key, $meta_value ) {
 	public function order_from_order_id( $order_id ) {
 		return wc_get_order( ( $order_id ) );
 	}
+
+	/**
+	 * Retrieve last WC refund from order ID.
+	 *
+	 * @param string $order_id WC Order ID.
+	 *
+	 * @return bool|WC_Order|WC_Order_Refund
```

Comment:
To ease the usage of this util function, returning either `null` or `WC_Order_Refund` seems a better option. 

Also, I think we may try our best to avoid using the direct SQL command. IMHO, `wc_get_orders` is still a good choice to use here. It provides [enough arguments (limit, orderby, and type)](https://github.com/woocommerce/woocommerce/wiki/wc_get_orders-and-WC_Order_Query) to get the same result and ensure the compatibility for [the future of a custom order table](https://developer.woocommerce.com/2022/01/17/the-plan-for-the-woocommerce-custom-order-table/).

Reply:
We may add tests to cover this util function too.

Reply:
Updated the PHPdoc comment and replaced DB query with `wc_get_orders`. I've originally added a [commit](https://github.com/Automattic/woocommerce-payments/pull/3965/commits/a3a8935376de6c56338f804b0164b2f917e049d1) that uses `wc_get_orders` and dropped it to use DB query.